doc/user_guide/user_guide.md

```diff
@@ -57,9 +58,14 @@ project root/
 * If the GitHub repository's releases page is not empty, the new release version must follow the versioning rules.
 It is not allowed to skip a version, to release the same version twice or to release a version that comes before the latest release.
 
-* The project must contain a `/.github/workflows/upload_release_asset.yml` file in the root directory. 
+* The project must contain a `/.github/workflows/upload_release_asset.yml` file in the root directory to upload release assets. 
 You can find examples [here](upload_release_asset_example.md).
```

Comment:
Always properly label links. Calling a link "here" is a convenient but terrible practice. :-p
Use a regular sentence instead an mark the part with the keyword as link.